editor/plugins/path_3d_editor_plugin.cpp

```diff
@@ -41,6 +41,9 @@
 #include "scene/gui/menu_button.h"
 #include "scene/resources/curve.h"
 
+// A flag to indicate if the camera is top view.
+bool is_top_view = false;
```

Comment:
This shouldn't be just a global boolean, it should be in the header

Reply:
@AThousandShips thanks for the feedback! This is my first ever contribution to c++ code, I'm working in JavaScript on daily basis 😅

Reply:
You forgot to remove this line :)